server/executor/functions/functions.go

```diff
@@ -0,0 +1,56 @@
+package functions
+
+import "fmt"
+
+type Invoker func(args ...Arg) string
+type Arg struct {
+	Value string
+	Type  string
+}
+
+type ArgConfig struct {
```

Comment:
Suggestion: since this is now a single field type, can it be converted to an alias?

```suggestion
type ArgTypes []string
```